docs-src/scim/index.rst

```diff
@@ -0,0 +1,97 @@
+==============================================
+SCIM API Client
+==============================================
+
+`SCIM API <https://api.slack.com/scim>`_ is a set of APIs for provisioning and managing user accounts and groups. SCIM is used by Single Sign-On (SSO) services and identity providers to manage people across a variety of tools, including Slack.
+
+`SCIM (System for Cross-domain Identity Management) <http://www.simplecloud.info/>`_ is supported by myriad services. It behaves slightly differently than other Slack APIs.
+
+Refer to `the API document <https://api.slack.com/scim>`_ for more details.
+
+SCIMClient
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+An OAuth token with `the admin scope <https://api.slack.com/scopes/admin>`_ is required to access the SCIM API.
+
+To fetch provisioned user data, you can use the ``search_users`` method in the client.
+
+.. code-block:: python
+
+    import os
+    from slack_sdk.scim import SCIMClient
+
+    client = SCIMClient(token=os.environ["SLACK_ORG_ADMIN_USER_TOKEN"])
+
+    response = client.search_users(
+        start_index=1,
+        count=100,
+        filter="""filter=userName Eq "Carly"""",
+    )
+    response.users  # List[User]
+
+Check `the class source code <https://github.com/slackapi/python-slack-sdk/blob/main/slack_sdk/scim/v1/user.py>`_ to know the structure of the ``user`` in the ``response.users``.
+
+Similarly, ``search_groups`` method is available and the ``Group`` object can be `like this <https://github.com/slackapi/python-slack-sdk/blob/main/slack_sdk/scim/v1/group.py>`_.
```

Comment:
```suggestion
Similarly, the ``search_groups`` method is available and the shape of the ``Group`` object can be `found here <https://github.com/slackapi/python-slack-sdk/blob/main/slack_sdk/scim/v1/group.py>`_.
```

Suggestion made assuming that the phrase 'can be like' was in reference to what the shape was.